package/autoyast2.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed May 27 14:03:50 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- When running an autoinstallation with the Online mediun, the
+  network configuration based on the profile can be written before
```

Comment:
```suggestion
  network configuration based on the profile needs to be activated before
```

Reply:
Well, I used "can" intentionally because it is only written in case of "setup_before_proposal" is true and needed in case it is not configured by linuxrc .. so was like an option, but not mandatory